pyproject.toml

```
@@ -1,104 +1,76 @@
 version = "2.41.0"
 description = "Text and image analysis of the digital collection (books, newspapers, periodicals, and images) at the National Library of Norway"
-authors = ["The Digital Humanities Lab at The National Library of Norway (NB) <dh-lab@nb.no>"]
-license = "MIT"
+authors = [
+    {name = "The Digital Humanities Lab at The National Library of Norway", email = "dh-lab@nb.no"}
+]
+license = {text = "MIT"}
 readme = "README.md"
-homepage = "https://www.nb.no/dh-lab/"
-documentation = "https://dhlab.readthedocs.io/en/stable/"
-repository = "https://github.com/NationalLibraryOfNorway/DHLAB"
+requires-python = "<3.13,>=3.10"
 classifiers = [
     "Programming Language :: Python :: 3",
     "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
 ]
-include = ["CHANGELOG.md", "LICENSE"]
-exclude = ["docs/", "tests/"]
-
-[tool.poetry.urls]
-"Bug Tracker" = "https://github.com/NationalLibraryOfNorway/DHLAB/issues"
-"Changelog" = "https://github.com/NationalLibraryOfNorway/DHLAB/blob/main/CHANGELOG.md"
-"Tutorials" = "https://nationallibraryofnorway.github.io/digital_tekstanalyse/tutorial.html"
-
-[tool.poetry.dependencies]
-python = ">=3.9"
-ipython = "^8.17.2"
-matplotlib = "^3.8.1"
-networkx = "^3.2.1"
-pandas = "^2.1.2"
-python-louvain = "^0.16"
-requests = "^2.31.0"
-seaborn = "^0.13.0"
-scipy = { version = "^1.11.3", python = ">=3.9,<3.13" }
-openpyxl = "^3.1.2"
-beautifulsoup4 = "^4.12.2"
-numpy = "^1.26.3"
-jinja2 = "^3.1.4"
-tqdm = "^4.66.6"
-
-[tool.poetry.group.docs.dependencies]
-Sphinx = "^7.2.6"
-furo = "^2023.9.10"
-myst-parser = "^2.0.0"
-readthedocs-sphinx-search = "^0.3.1"
-sphinx-copybutton = "^0.5.2"
-sphinx-togglebutton = "^0.3.2"
-sphinx_design = "^0.5.0"
-sphinx_inline_tabs = "^2023.4.21"
-mkdocs = "^1.5.2"
-mkdocstrings = {extras = ["python"], version = "^0.22.0"}
-mkdocs-material = "^9.1.21"
-mkdocs-section-index = "^0.3.5"
-mkdocs-autorefs = "^0.5.0"
-sphinx-autodoc2 = "^0.5.0"
-sphinx-book-theme = "^1.1.2"
-linkify-it-py = "^2.0.3"
-
-
-[tool.poetry.group.dev.dependencies]
-ipykernel = "^6.26.0"
-pytest = "^7.4.3"
-pytest-cov = "^4.1.0"
-mypy = "^1.6.1"
-black = "^23.10.1"
-isort = "^5.12.0"
-ruff = "^0.3.0"
-
-[tool.poetry.group.test.dependencies]
-mypy = "^1.6.1"
-pytest = "^7.4.3"
-pytest-cov = "^4.1.0"
-pytest-mock = "^3.12.0"
-
-[tool.poetry-dynamic-versioning]
-enable = false
-bump  = false
-vcs = "git"
-format = "v{base}{stage}{revision}"
-
-
-[tool.poetry-dynamic-versioning.substitution]
-files = ["dhlab/_version.py"]
-
-[build-system]
-requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
-build-backend = "poetry_dynamic_versioning.backend"
+dependencies = [
+    "ipython>=8.17.2",
+    "matplotlib>=3.8.1",
+    "networkx>=3.2.1",
+    "pandas>=2.2.3",
+    "python-louvain>=0.16",
+    "requests>=2.31.0",
+    "seaborn>=0.13.2",
+    "scipy==1.14",
+    "openpyxl>=3.1.2",
+    "beautifulsoup4>=4.12.2",
+    "jinja2>=3.1.4",
+    "tqdm>=4.66.6",
+    "numpy==1.26",
```

```suggestion
    "scipy>=1.14",
    "openpyxl>=3.1.2",
    "beautifulsoup4>=4.12.2",
    "jinja2>=3.1.4",
    "tqdm>=4.66.6",
    "numpy>=1.26",
```

It would be nice not to have a hard pin on old versions of numpy and scipy

---

I absolutely agree - but ref. the comment above (https://github.com/NationalLibraryOfNorway/DHLAB/pull/236/files#r1927195453), the library breaks with higher versions.